pontoon/base/models.py

```diff
@@ -475,7 +470,7 @@ def available(self):
         """
         Available locales have at least one TranslatedResource defined.
         """
-        return self.filter(translatedresources__isnull=False).distinct()
+        return self.filter(project_locale__isnull=False).distinct()
```

Comment:
I don't understand this change. Can you explain it, or add a comment?

Reply:
... also noticed that the comment above this is not matching what the code does anymore

Reply:
This was explained in https://github.com/mozilla/pontoon/pull/705/commits/d0847e7f0be458019ae2b147c21d8c3c3424ace7, but poorly. :) I noticed the query was slow and fixed it by changing it. Instead of showing all locales that have at least one resource of at least one project enabled (synced), we'd show all locales that have at least one project enabled (even if it's not synced yet).

Now I restored the behaviour so it's exactly the same as originally, but the query is much faster. So I'm leaving the comment unchanged.

If we decide to make the change on which locales to list, we can do that in a followup commit.